packages/components/src/dropdown-menu-v2/index.tsx

```diff
@@ -215,6 +216,9 @@ export const DropdownMenuV2 = Object.assign(
 		Group: Object.assign( DropdownMenuGroup, {
 			displayName: 'DropdownMenuV2.Group',
 		} ),
+		GroupLabel: Object.assign( DropdownMenuGroupLabel, {
```

Comment:
Is this worth adding to the component README too?

BTW, unrelated, but I spotted a few things in that README:

* Not all components follow the dot notation. Example: `DropdownMenuGroup`.
* We seem to document a `DropdownMenuSeparatorProps` component, perhaps a remnant from v1? https://github.com/WordPress/gutenberg/blob/d8160d86566d1a0e9a3db592131136ef99cc816a/packages/components/src/dropdown-menu-v2/README.md#L328

There might be more inconsistencies, it's worth doing an audit.

Reply:
Nice find! I fixed those instances.

We will definitely do a documentation overhaul once the component is more stable — I figured that there's not much point at the moment, given that I'm planning on changing the APIs by switching to more granular subcomponents